code/lib/types/src/modules/core-common.ts

```diff
@@ -335,6 +335,15 @@ export interface TestBuildConfig {
   test?: TestBuildFlags;
 }
 
+type Tag = string;
+
+export interface TagOptions {
+  excludeFromSidebar: boolean;
+  excludeFromAutodocs: boolean;
+}
+
+export type TagsOptions = Record<Tag, Partial<TagOptions>>;
```

Comment:
Shouldn't we move this to `@storybook/types`? Then it can be reused in `code/addons/docs/src/typings.d.ts` and we don't have to type it with `any` there.

Reply:
This is already in `@storybook/types`

Reply:
Fixed